src/main/java/uk/gov/hmcts/darts/common/task/ProcessDailyListAutomatedTask.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.darts.common.task;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.LocalDateTime;
+
+@Slf4j
+public class ProcessDailyListAutomatedTask implements AutomatedTask {
+
+    public static final String PROCESS_DAILY_LIST_TASK_NAME = "ProcessDailyList";
+
+    @Override
+    public String getTaskName() {
+        return PROCESS_DAILY_LIST_TASK_NAME;
+    }
+
+    @Override
+    public void run() {
+        log.info("Task: {} running at: {} {}", getTaskName(),
```

Comment:
Leave a TODO here to indicate this is just a temporary implementation. Ideally reference the ticket number that will implement this.